### ISSUE TITLE
Feature: Windows ARM64 support

### ISSUE THREAD

**Issue by mhoogenbosch** [jimmyeao/TEAMS2HA#65]
**Is your feature request related to a problem? Please describe.**
I just received my new Surface Pro 11th gen. This has a ARM64 processor architecture. Native compiled code runs better, and mostly faster.

**Describe the solution you'd like**
I don't know how to, but i'd think there must be a way to compile it for ARM based systems. 

**Describe alternatives you've considered**
The .exe now get's emulated, so it does work, but not as smooth as it could

**Additional context**

![image](https://github.com/user-attachments/assets/cb8d581a-7cc5-4ae5-9e73-a02587a24f17)

**Comment by mhoogenbosch** [comment]
Does a closed Enhancement stay on the backlog?

**Comment by jimmyeao** [comment]
Yes, its still on my to do list. I need to get an Arm device to test on, hopefully Dell will send me an eval one if I ask nicely :)

**Comment by mhoogenbosch** [comment]
> Yes, its still on my to do list. I need to get an Arm device to test on, hopefully Dell will send me an eval one if I ask nicely :)

If I can help with anything I'm more than happy to do so.

**Comment by jimmyeao** [comment]
Hi, Ive just compiled this on an Arm machine (Dell finally came through for me!) and added to the latest release - download the .zip, extract and run setup, should be Arm native

**Comment by mhoogenbosch** [comment]
Whoop whoop, before I updated:

![image](https://github.com/user-attachments/assets/bd5b7402-0384-4401-893a-a1486395d7ed)

and now:

![image](https://github.com/user-attachments/assets/1ea33d46-02f3-4963-ab2b-e486d29dd2f2)

I did have to remove it before installing it, because it mentioned it couldn't start the install because it was already installed on a different location. After removing and installing, it worked and kept my previous settings.

**Comment by jimmyeao** [comment]
Awesome, thanks for the feedback and testing :)

**Comment by jimmyeao** [comment]
Awesome, thanks for the feedback and testing :)
